tests/Action/SendEmailActionTest.php

```diff
@@ -25,38 +25,29 @@
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
 
-class SendEmailActionTest extends TestCase
+final class SendEmailActionTest extends TestCase
 {
     /**
      * @var MockObject&UrlGeneratorInterface
      */
-    protected MockObject $urlGenerator;
+    private MockObject $urlGenerator;
 
     /**
      * @var MockObject&UserManagerInterface
      */
-    protected MockObject $userManager;
+    private MockObject $userManager;
 
     /**
      * @var MockObject&MailerInterface
      */
-    protected MockObject $mailer;
+    private MockObject $mailer;
 
     /**
      * @var MockObject&TokenGeneratorInterface
      */
-    protected MockObject $tokenGenerator;
+    private MockObject $tokenGenerator;
 
-    protected int $resetTtl;
-
-    protected string $fromEmail;
-
-    protected string $template;
```

Comment:
This was removed, and still used

Reply:
It was only used in the setUp function, fixed now.